bvbabel/vmr.py

```diff
@@ -104,6 +108,9 @@ def read_vmr(filename):
         # left-right convention, the reference space (e.g. Talairach after
         # normalization) and voxel resolution.
 
+        # Early return for headerless data (e.g. for v16 files)
+        if 'File version' not in header: return header, data_img
```

Comment:
Avoid single line flow controls to comply with the style in the rest of the project.